build on ios device found error

./../go/pkg/mod/github.com/99designs/keyring@v1.1.6/keychain.go:55:8: query.SetMatchSearchList undefined (type keychain.Item has no field or method SetMatchSearchList)
../../go/pkg/mod/github.com/99designs/keyring@v1.1.6/keychain.go:55:28: undefined: keychain.NewWithPath
../../go/pkg/mod/github.com/99designs/keyring@v1.1.6/keychain.go:115:34: undefined: keychain.Keychain
../../go/pkg/mod/github.com/99designs/keyring@v1.1.6/keychain.go:124:12: queryItem.SetMatchSearchList undefined (type keychain.Item has no field or method SetMatchSearchList)
../../go/pkg/mod/github.com/99designs/keyring@v1.1.6/keychain.go:136:8: kcItem.SetAccess undefined (type keychain.Item has no field or method SetAccess)
../../go/pkg/mod/github.com/99designs/keyring@v1.1.6/keychain.go:146:9: undefined: keychain.Keychain
../../go/pkg/mod/github.com/99designs/keyring@v1.1.6/keychain.go:166:9: kcItem.UseKeychain undefined (type keychain.Item has no field or method UseKeychain)
../../go/pkg/mod/github.com/99designs/keyring@v1.1.6/keychain.go:181:9: kcItem.SetAccess undefined (type keychain.Item has no field or method SetAccess)
../../go/pkg/mod/github.com/99designs/keyring@v1.1.6/keychain.go:181:21: undefined: keychain.Access
../../go/pkg/mod/github.com/99designs/keyring@v1.1.6/keychain.go:267:36: undefined: keychain.Keychain
../../go/pkg/mod/github.com/99designs/keyring@v1.1.6/keychain.go:181:21: too many errors

--- a/go.mod
+++ b/go.mod
@@ -3,11 +3,11 @@ module github.com/99designs/keyring
 go 1.14
 
 require (
+	github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4
 	github.com/danieljoos/wincred v1.0.2
 	github.com/dvsekhvalnov/jose2go v0.0.0-20200901110807-248326c1351b
 	github.com/godbus/dbus v0.0.0-20190726142602-4481cbc300e2
 	github.com/gsterjov/go-libsecret v0.0.0-20161001094733-a6f4afe4910c
-	github.com/keybase/go-keychain v0.0.0-20190712205309-48d3d31d256d
 	github.com/kr/pretty v0.1.0 // indirect
 	github.com/mtibben/percent v0.2.1
 	github.com/stretchr/objx v0.3.0 // indirect
@@ -17,5 +17,3 @@ require (
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
-
-replace github.com/keybase/go-keychain => github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4

--- a/keychain.go
+++ b/keychain.go
@@ -7,7 +7,7 @@ import (
 	"errors"
 	"fmt"
 
-	gokeychain "github.com/keybase/go-keychain"
+	gokeychain "github.com/99designs/go-keychain"
 )
 
 type keychain struct {

